test/common_utils.py

```diff
@@ -0,0 +1,33 @@
+import uuid
+
+
+def generate_unique_name(prefix, timestamp_string) -> str:
+    name = prefix + "_" + str(uuid.uuid4()) + "_" + timestamp_string
+    return name
+
+
+def run_proc(conn_cnx, sql) -> str:
+    with conn_cnx() as cnx:
+        cur = cnx.cursor()
+        result = cur.execute(sql).fetchone()
+        return result
+
+
+def delete_list_of_labels(conn_cnx, sql):
+    print(f"[INFO] SQL in delete function: {sql}")
+
+    with conn_cnx() as cnx:
+        cur = cnx.cursor()
+        for name in cur.execute(sql).fetchall():
+            delete_label_statement = f"call ADMIN.DELETE_LABEL('{name[0]}');"
+            assert "done" in str(
+                run_proc(conn_cnx, delete_label_statement)
+            ), "Stored procedure output does not match expected result!"
```

Comment:
This is a little wonky by mixing both test cleanup logic with OpsCenter "application" logic.

What about making API methods that could do things like create, list, or delete labels (calling the appropriate SQL in those methods), and then repurpose this method to just be a "teardown" method which calls those new methods? e.g.

```
def teardown(conn_cnx):
    ...
    conn = conn_cnx()
    for name in list_labels(conn):
        delete_label(conn, name)
```

I think this will have the bonus of making your test code much more terse (less SQL) and easier to extend.

Reply:
Josh, the reason I'm querying internal table for labels is because I'm protecting against the case where due to a bug of some sort, negative test did not fail as expected and label got created. In this case, I'm finding every label that was created in a test session and deleting it as there is no other method to "list labels" (or I'm not aware of it)

As to "less SQL" code ... Ryan and I wanted to make this as simple as possible: when test fails, you see exactly which SQL statement failed without any extra debugging + easy to add specifically SQL tests... Dunno ... let's talk. Maybe I'm missing something important. Thanks for the review, I found it very helpful!